spinn_front_end_common/interface/buffer_management/buffer_manager.py

```diff
@@ -177,6 +177,7 @@ def load_initial_buffers(self):
             total_data, "Loading buffers ({} bytes)".format(total_data))
         for vertex in self._sender_vertices:
             for region in vertex.get_regions():
+                vertex.rewind(region)
```

Comment:
This worries me a lot. Rewinding the vertex every single time forces the buffered manager to be a "reset, try again" mode. Whereas multiple runs is meant to be more of a pause and resume mode. Therefore Ive extracted the rewind here and made it a seperate method which is called rewind, and we will need to think how this is powered from the top level of PyNN, given that there is only reset and run as functions in the PyNN offical language.